packages/card/src/Card/types.ts

```diff
@@ -0,0 +1,28 @@
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+export const ContentStyle = {
+    None: 'none',
+    Clickable: 'clickable',
+} as const;
+
+export type ContentStyle = typeof ContentStyle[keyof typeof ContentStyle];
+
+
+
+export interface CardProps extends HTMLElementProps<'div'> {
+    /**
+     * Determines whether the Card should be styled as clickable.
+     *
+     * Defaults to `'clickable'` (when a valid `onClick` handler or `href` link is provided
+     *
+     * @default 'clickable' | 'none'
+     */
+    contentStyle?: ContentStyle;
+
+    /**
+     * Determines whether or not the component will appear in dark mode.
+     *
+     * @default false
+     */
+    darkMode?: boolean;
+}
```

Comment:
I know you just copied and pasted this, but we have a DarkModeProps we can extend instead of explicitly defining it here.

Reply:
will update!